application/common/models/User.php

```diff
@@ -1274,4 +1276,33 @@ public static function getActiveUnlockedUsers()
     {
         return User::findAll(['active' => 'yes', 'locked' => 'no']);
     }
+
+    public static function exportToSheets()
+    {
+        Yii::info([
+            'action' => 'export to Google Sheets',
+            'status' => 'start',
+        ]);
+
+        $googleSheetsClient = new Sheets([
+            'applicationName' => \Yii::$app->params['google']['applicationName'],
+            'jsonAuthFilePath' => \Yii::$app->params['google']['jsonAuthFilePath'],
+            'jsonAuthString' => \Yii::$app->params['google']['jsonAuthString'],
+            'delegatedAdmin' => \Yii::$app->params['google']['delegatedAdmin'],
+            'spreadsheetId' => \Yii::$app->params['google']['spreadsheetId'],
+        ]);
+
+        $activeUsers = User::find()->where(['active' => 'yes', 'locked' => 'no'])->all();
```

Comment:
```suggestion
        $activeUsers = User::find()->where(['active' => 'yes'])->all();
```

Reply:
I think we can/should include locked users since that can just be a temporary thing.